efs.tf

```diff
@@ -0,0 +1,5 @@
+# Get list of EFS ids
+data "aws_efs_file_system" "by_id" {
+  count          = "${length(keys(var.efs_id_mount_point))}"
```

Comment:
I think `mount_point` is misleading. These are just `efs_id`s. 

https://www.terraform.io/docs/providers/aws/r/efs_mount_target.html